src/assets/styles/base/_markdown.scss

```diff
@@ -136,7 +136,13 @@
 
   h1 {
     @include h(1);
+    font-family: $fleuron-reg;
```

Comment:
@alecmarcus is this still where we want to put type styles for entries? I'm a little unsure of where to put what/how all of the stylesheets are referencing each other

Reply:
yes! this is a global stylesheet, for all rendered markdown across the site